charts/app-config/values-production.yaml

```diff
@@ -1808,6 +1819,7 @@ govukApplications:
         - name: PUMA_TIMEOUT
           value: "300"
       cronTasks: []
+      appEnabled: false
   - name: licensify
```

Comment:
Could you double check `licensify`? I can't see it being disabled with `appEnabled: false` but I seem to remember it had a different key (it may have changed)